TrackerConditions/src/TrackerDAQConditions.cc

```diff
@@ -8,6 +9,10 @@ namespace mu2e {
     uint16_t dracChannel = packetId & 0xFF; 
```

Comment:
Will these shifts and masks ever change?  Are the common to other subsystems?

Reply:
They may change when we finalize the packet format, there's one free bit here we could use for a flag if we need. And this is tracker specific.